File: Modul-3/C/PRAK301-2310817110006-Adrian_Bintang_Saputera.c
#include <stdio.h>

int main()
{
  int angka1, angka2;
  printf("Masukkan 2 angka acak: ");
  scanf("%d %d", &angka1, &angka2);

  if (angka1 <= angka2)
  {
    printf("Urutan terkecil ke terbesar: %d %d\n", angka1, angka2);
  }
  else
  {
    printf("Urutan terkecil ke terbesar: %d %d\n", angka2, angka1);
  }

  return 0;
}

File: Modul-3/Python/PRAK301-2310817110006-Adrian_Bintang_Saputera.py
angka1, angka2 = map(int, input("Masukkan 2 angka acak: ").split())

if angka1 <= angka2:
    print(f"Urutan terkecil ke terbesar: {angka1} {angka2}")
else:
    print(f"Urutan terkecil ke terbesar: {angka2} {angka1}")